CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ and this repository adheres to [Semantic Versioning](https://semver.org/spec/v2.
 
 ## Unreleased
 
+tbd 
+
+## v1.30.2 - 2024-02-15
+
+- Several dependency upgrades.
```

Comment:
I think this isn't needed. If we start doing this then we should add this sentence to all changelog entries.